lux/config/config.exs

```diff
@@ -6,6 +6,10 @@ config :lux, :open_ai_models,
   default: "gpt-4o-mini",
   smartest: "gpt-4o"
 
+# API Keys
+config :lux, :api_keys,
+  discord: System.get_env("DISCORD_BOT_TOKEN", "test_bot_token")
```

Comment:
Please check how we set config values at runtime in `runtime.exs` and using dotenvy to load values from different config files and the system as a fallback. This should move to `runtime.exs`